enterprise/models.py

```diff
@@ -67,7 +67,7 @@ class Meta(object):
     name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Enterprise Customer name."))
     active = models.BooleanField(default=True)
     history = HistoricalRecords()
-
+    identity_provider = models.SlugField(null=True, blank=True, default=None)
```

Comment:
I know that, One EC can be associated with One SAML-IDP. 
@mattdrayer @saleem-latif but one thing in confusing me, that is  **You can add multiple EC(s) against a single SAML-IDP.** If it is the required behavior then how can we fetch the particular EC against the tpa-hint (identity-provider) ? 

<img width="1258" alt="screen shot 2016-11-22 at 12 24 55 pm" src="https://cloud.githubusercontent.com/assets/7334669/20514611/c65bccce-b0ae-11e6-86c6-6a4e285a4f28.png">


`redirected_url?tpa_hint=saml-ubc`